File: server/default_codes/driver_codes/driver.cpp
#include <bits/stdc++.h>
using namespace std;

// -----------------------------
// Basic Linked List Node
// -----------------------------
struct ListNode {
        int val;
        ListNode *next;
        ListNode(int x) : val(x), next(nullptr) {
        }
};

// -----------------------------
// Basic Binary Tree Node
// -----------------------------
struct TreeNode {
        int val;
        TreeNode *left;
        TreeNode *right;
        TreeNode(int x) : val(x), left(nullptr), right(nullptr) {
        }
};

// class Solution {
// public:
//     void solve( ) {
//         // Write your code here
//     }
// };

int main( ) {
        Solution sol = Solution();
        sol.solve();
        return 0;
}

File: server/default_codes/driver_codes/driver.java
import java.util.*;

// -----------------------------
// Basic Linked List Node
// -----------------------------
class ListNode {
    int val;
    ListNode next;
    ListNode(int x) {
        val = x;
        next = null;
    }
}

// -----------------------------
// Basic Binary Tree Node
// -----------------------------
class TreeNode {
    int val;
    TreeNode left;
    TreeNode right;
    TreeNode(int x) {
        val = x;
        left = right = null;
    }
}

// class Solution {
//     public void solve() { }
// }

public class Main {
    public static void main(String[] args) {
        Solution sol = new Solution();
        sol.solve();
    }
}
